packages/js/src/core/sign-out.ts

```diff
@@ -0,0 +1,15 @@
+import { conditional } from '@silverhand/essentials';
+import qs from 'query-string';
+
+export const generateSignOutUri = (
+  endSessionEndpoint: string,
+  idToken: string,
+  postLogoutRedirectUri: string | null
+) => {
+  const url = `${endSessionEndpoint}?${qs.stringify({
+    id_token_hint: idToken,
+    ...conditional(postLogoutRedirectUri && { post_logout_redirect_uri: postLogoutRedirectUri }),
```

Comment:
will `qs` handle this?

Reply:
No, althought qs will not append null value, it will still append the param name, which ends up like this:
`id_token_hint=id_token&post_logout_redirect_uri`

Reply:
OK, query-string does have an option `skipNull` that does exactly the same thing.
Code updated.